core/sds-test-aspect-models/src/main/resources/invalid/bamm_2_0_0/io.openmanufacturing.test/1.0.0/InvalidPreferredNameDatatype.ttl

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2021 Robert Bosch Manufacturing Solutions GmbH
```

Comment:
Why did you not add this test model for BAMM version 1.0.0?

Reply:
The current implementation of the AspectModelResolver that also used in the test suite will automatically upgrade BAMM 1.0.0-based models to 2.0.0 so the shapes won't match the model. This should probably be fixed but is out of scope for this PR IMO.